packages/EasyCore/src/Doctrine/Events/EntityCreatedEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Events;
+
+use EonX\EasyCore\Interfaces\DatabaseEntityInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+final class EntityCreatedEvent extends Event
+{
+    /**
+     * @var string
+     */
+    public const NAME = 'entity.created';
```

Comment:
We don't need this const anymore, right?

Reply:
Done